File: Arcade/Intro/01_The_Journey_Begins/03_checkPalindrome/main.java
boolean checkPalindrome(String inputString) {
    int l = inputString.length();
    if (l < 2)
        return true;
    for (int i = 0; i <= (l / 2); i++) {
        if (!(inputString.substring(i, i + 1).equals(inputString.substring(l - 1 - i, l - i)))) {
            return false;
        }
    }
    return true;
}

File: Arcade/Intro/01_The_Journey_Begins/03_checkPalindrome/main.ts
function checkPalindrome(inputString: string): boolean {
    let l: number = inputString.length;
    if (l < 2) return true;
    for (let i: number = 0; i <= (l / 2); i++) {
        if (!(inputString.substring(i, i + 1) === inputString.substring(l - 1 - i, l - i)))return false;
    }
    return true;
}
